strategy-offline-fallback/controlled.js

```diff
@@ -0,0 +1,18 @@
+// To be able to provide fallbacks since the beginning we can only load the
+// image once we know the SW is ready. In addition, the SW should start
+// intercepting ASAP.
```

Comment:
Please don't use abbreviations in the comments unless necessary. This particular block is littered with acronyms and makes things hard to read